src/bp/lang-server/api.ts

```diff
@@ -124,13 +124,29 @@ export default async function(options: APIOptions) {
       domain: options.service.domain,
       readOnly: options.readOnly,
       languages: options.service
-        .listModels()
+        .listFastTextModels()
         .filter(x => x.loaded)
         .map(x => x.name)
     })
   })
 
   app.post('/vectorize', waitForServiceMw, async (req, res, next) => {
+    try {
+      const input = req.body.input
+      const lang = req.body.lang || 'en'
+
+      if (!input || !isString(input)) {
+        throw new BadRequestError('Param `input` is mandatory (must be an array of strings)')
+      }
+
+      const [result, tokens] = await options.service.vectorize(input, lang)
+      res.json({ input: input, language: lang, vectors: result, tokens: tokens })
```

Comment:
When the parameters are self-telling, you can avoid writing them twice, ex: ({ input, language, vectors, tokens }). You can also rename your previous parameter "lang" to "language" to avoid the language: lang conversion. 

Also, you can deconstruct your call with [vectors, tokens] instead of the placeholder result